install.sh

```diff
@@ -33,6 +33,30 @@ MAGENTA='\e[1;35m';
 RESET='\e[0m'
 
 echo -e "${GREEN}Welcome to Chromebrew!${RESET}\n"
+
+# load system info
+if [ -f /etc/lsb-release ]; then
+  source /etc/lsb-release
+else
+  echo -e "${YELLOW}Unable to detect system information, installation will continue.${RESET}"
+  CHROMEOS_RELEASE_VERSION=0.0-00000
```

Comment:
You could probably use just `CHROMEOS_RELEASE_VERSION=0`. :)

Reply:
Done